locale/templates/settings.ftl

```diff
@@ -479,7 +479,7 @@ row-defaults-status = None
 
 rk-header = Recovery key
 rk-enabled = Enabled
-rk-not-set = Not set
+rk-not-set = Not Set
```

Comment:
If the change is a must, then we need a new string ID otherwise, reusing the same ID will not trigger Pontoon as a new or revised string - this is the case for those locales already translated the strings, including en-CA and en-GB.

Reply:
we don't want to trigger any changes in pontoon for this. i made it so that our local env will match production for en locales